hail_scripts/v01/utils/elasticsearch_utils.py

```diff
@@ -155,7 +167,7 @@ def generate_vds_make_table_arg(field_path_to_field_type_map, is_split_vds=True)
         # drop the 'v', 'va' root from key-table key names
         key = _field_path_to_elasticsearch_field_name(field_path)
         expr = "%s = %s" % (key, ".".join(field_path))
-        if is_split_vds and field_type.startswith("Array"):
+        if is_split_vds and isinstance(field_type, basestring) and field_type.startswith("Array"):
```

Comment:
Out of curiosity, why is the `isinstance(field_type, basestring)` check needed now?

Reply:
for nested fields, field_type is dict which breaks field_type.startswith("Array")

Reply:
Doh, I should have realized from the `if isinstance(field_type, dict)` above.

The docstring for the `field_path_to_field_type_map` argument claims that values are strings.

Reply:
oops, fixed